addons/l10n_ar_withholding/models/account_chart_template.py

```diff
@@ -0,0 +1,43 @@
+from odoo import models
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+
+    _inherit = 'account.chart.template'
+
+    # ar base
+    @template('ar_base', 'account.account')
+    def _get_ar_base_withholding_account_account(self):
+        return self._parse_csv('ar_base', 'account.account', module='l10n_ar_withholding')
+
+    # ri chart
+    @template('ar_ri', 'account.tax.group')
+    def _get_ar_ri_withholding_account_tax_group(self):
+        return self._parse_csv('ar_ri', 'account.tax.group', module='l10n_ar_withholding')
+
+    @template('ar_ri', 'account.tax')
+    def _get_ar_ri_withholding_account_tax(self):
+        additional = self._parse_csv('ar_ri', 'account.tax', module='l10n_ar_withholding')
+        self._deref_account_tags('ar_ri', additional)
+        self._deref_account_tags('ar_ex', additional)
+        return additional
+
+    # ex chart
+    @template('ar_ex', 'account.tax.group')
+    def _get_ar_ex_withholding_account_tax_group(self):
+        return self._parse_csv('ar_ex', 'account.tax.group', module='l10n_ar_withholding')
+
+    @template('ar_ex', 'account.tax')
+    def _get_ar_ex_withholding_account_tax(self):
+        additional = self._parse_csv('ar_ex', 'account.tax', module='l10n_ar_withholding')
+        self._deref_account_tags('ar_ex', additional)
+
+    def _load(self, template_code, company, install_demo):
+        res = super()._load(template_code, company, install_demo)
+
+        # If Responsable Monotributista remove the default purchase tax
+        if template_code in ['ar_base', 'ar_ri', 'ar_ex']:
+            company.l10n_ar_tax_base_account_id = self.ref('base_tax_account')
```

Comment:
check if there is a better way to do it. Check here https://github.com/odoo/odoo/blob/084408a9bb26dcc1ac54986802b08638c72b6235/addons/l10n_be/models/template_be.py#L20

Reply:
DONE!_